src/js/start.js

```diff
@@ -85,7 +85,9 @@ $(function () {
     $(this).attr('tabindex', '-1');
   });
 
-  toggleFormInput($('.usa-show_password'), 'Show Password', 'Hide Password');
-  toggleFormInput($('.usa-show_multipassword'), 'Show my typing', 'Hide my typing');
-  validator($('.js-validate_password'));
+  if (/form-controls/.test(window.location.pathname)) {
```

Comment:
I agree @msecret good catch. This should be targeting `class`es or `id`s related to `form-controls`.